images/encryption/utils.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
any reason this one function shouldn't just go into your utils package (next file)? or conversely any reason the utility functions in the separate `encryption/utils` pkg shouldn't just end up in this file?

Reply:
The function  `Uint64ToStringArray` in this file only has one user in `images/encryption/gpg.go` and could either move in there or we can merge it into `images/encryption/utils/utils.go` where there are more crypto related functions, though. Either way is fine by me. If you don't have a preference I'd probably move it into gpg.go.